gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -262,6 +264,20 @@ private boolean isDirectPathEnabled(String serviceAddress) {
     return false;
   }
 
+  private boolean isDirectPathXdsUsed() {
```

Comment:
Can we test this behavior in unit tests?

Reply:
Added.